### ISSUE TITLE
Weird `}` in gitlab output

### ISSUE THREAD

**Issue by glensc** [orta/danger-plugin-yarn#48]
<img width="414" alt="image" src="https://user-images.githubusercontent.com/199095/159357079-6f36f422-de92-47ca-b663-476ebb34ac74.png">

Seems to be present after each package name.

Here's copy of full HTML:
- [danger.html](https://github.com/orta/danger-plugin-yarn/files/8319028/danger.html.txt)

**Comment by glensc** [comment]
@orta can you make a new release, this time actually containing the fix?

See https://github.com/orta/danger-plugin-yarn/pull/45#issuecomment-1074393518

Preferably with some automation for transparency to the end users:
- https://github.com/orta/danger-plugin-yarn/pull/51

**Comment by glensc** [comment]
The current workaround, use any version before:
- https://github.com/orta/danger-plugin-yarn/pull/44

That appears to be 1.3.3

**Comment by glensc** [comment]
@orta ping. seems easy fix just make new release from the default branch

**Comment by orta** [comment]
Yep, deployed as 1.5.0

**Comment by glensc** [comment]
Looks like 1.5.0 update is breaking change. No longer can run jest

```

    Details:

    /Users/glen/Library/Caches/glen/node_modules/danger-plugin-dangered/node_modules/danger-plugin-yarn/node_modules/node-fetch/src/index.js:9
    import http from 'node:http';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

```

**Comment by glensc** [comment]
Apparently broken with 9e16443b143ef299cb83a784c8fb7c4b651754f1 commit

Unclear what's the commit purpose,
Why no Pull Request? Why no explanation what is the problem to be solved?

Perhaps this repo should be moved to a danger organization, so the community can deal with the maintenance?

**Comment by orta** [comment]
Broke by https://github.com/orta/danger-plugin-yarn/commit/714f03b96894b7e01c604ccee3d74da60da656b3 and the commit message says what it does - typescript did not build on master when I cloned it

**Comment by orta** [comment]
Fixed in 1.5.1, and no, I'm not really into moving to the danger org - that's not really for plugins

**Comment by glensc** [comment]
Confirming fixed with 1.5.1. Thanks!
